luigi/contrib/spark.py

```diff
@@ -168,15 +192,13 @@ def archives(self):
     def hadoop_conf_dir(self):
         return configuration.get_config().get(self.spark_version, "hadoop-conf-dir", None)
 
-    def get_environment(self):
```

Comment:
This seems to be a breaking change though.

Reply:
@honnix 
it's an internal method used by `SparkSubmitTask` in `program_environment` only. 
But I agree that it's better not to change

Reply:
@honnix 
done

Reply:
Thanks. It's unfortunate we leaked this method as 'public'.